### ISSUE TITLE
Execute Anonymous Error 

### ISSUE THREAD

**Issue by naveenila1221** [xjsender/haoide-vscode#10]
Hi 

I executed below code in VS Code anonymous window: 

`List<String> tempList = new List<String>();
tempList.add('aaaaa');
tempList.add('bbbbb');
tempList.add('ccccc');
System.debug(tempList);`

Getting an error message : 

500 - "<?xml version=\"1.0\" encoding=\"UTF-8\"?><soapenv:Envelope xmlns:soapenv=\"http://schemas.xmlsoap.org/soap/envelope/\"><soapenv:Body><soapenv:Fault><faultcode>soapenv:Client</faultcode><faultstring>Unexpected element {}String during simple type deserialization</faultstring></soapenv:Fault></soapenv:Body></soapenv:Envelope>"

**Comment by xjsender** [comment]
@naveenila1221 , yes, this is a bug, I still not find the suitable lib for escape the apex code, just like ``quoteattr`` in the ``python``, thanks for you response, it will be fixed in the next release.
